app/views/layouts/application.html.haml

```diff
@@ -1,58 +1,70 @@
 !!! 5
+<!DOCTYPE html>
+<!--[if IE]><![endif]-->
+<!--[if lt IE 7 ]>
+<html lang="en" class="ie6 no-js">    <![endif]-->
+<!--[if IE 7 ]>
+<html lang="en" class="ie7 no-js">    <![endif]-->
+<!--[if IE 8 ]>
+<html lang="en" class="ie8 no-js">    <![endif]-->
+<!--[if IE 9 ]>
+<html lang="en" class="ie9 no-js">    <![endif]-->
+<!--[if (gt IE 9)|!(IE)]><!-->
+<html lang="en" class="no-js"><!--<![endif]-->
 
-%html.js{lang: "en-au"}
-  = render 'application/head',  title: (content_for?(:title) ? "#{yield(:title)} | GOV.AU" : 'GOV.AU' )
-  %body
-    %div.skip-to
-      %a{href: "#content"}
-        Skip to main content
-    %header{role: 'banner'}
-      = render 'application/signed_in_nav' if user_signed_in?
-      %section.prerelease-govau--header
-        - if content_for?(:header)
-          = yield(:header)
-        - else
-          .wrapper
-            %a.govau--logo{href: "/"}
-              = image_tag 'icons/logo-govau-full-2x.png', width: '270'
-              .brandbar.default
-                .b1
-                .b2
-            = #render partial: 'feedback_and_menu'
+= render 'application/head',  title: (content_for?(:title) ? "#{yield(:title)} | GOV.AU" : 'GOV.AU' )
+%body
+  %div.skip-to
+    %a{href: "#content"}
+      Skip to main content
+  %header{role: 'banner'}
+    = render 'application/signed_in_nav' if user_signed_in?
+    %section.prerelease-govau--header
+      - if content_for?(:header)
+        = yield(:header)
+      - else
+        .wrapper
+          %a.govau--logo{href: "/"}
+            = image_tag 'icons/logo-govau-full-2x.png', width: '270'
+            .brandbar.default
+              .b1
+              .b2
+          = #render partial: 'feedback_and_menu'
 
-          %nav.breadcrumbs--inverted{'aria-label' => 'breadcrumb'}
-            .wrapper
-              = breadcrumbs style: :ul, class: nil
+      %nav.breadcrumbs--inverted{'aria-label' => 'breadcrumb'}
+        .wrapper
+          = breadcrumbs style: :ul, class: nil
```

Comment:
This is now outside the `if content_for?(:header)` condition which means the breadcrumbs will be rendered on the home page. I believe it was part of the new banner designs not to have the breadcrumbs on the home page. Was there a specific reason to add them back?